elytron-oidc-client/subsystem/src/test/java/org/wildfly/extension/elytron/oidc/OidcActivationProcessorTest.java

```diff
@@ -0,0 +1,205 @@
+package org.wildfly.extension.elytron.oidc;
```

Comment:
Just a very minor comment. This file is missing a copyright header. This can be copied/pasted from any other java file.